forage-android/src/main/java/com/joinforage/forage/android/network/data/CapturePaymentRepository.kt

```diff
@@ -160,6 +180,12 @@ internal class CapturePaymentRepository(
             )
         )
 
+        // END METRICS TRACKING UPON SUCCESSFUL FULL TRIP
+        // ------------------------------------------------------
+        responseMonitor.end()
+        responseMonitor.logResult()
+        // ------------------------------------------------------
+
         return paymentService.getPayment(paymentRef = paymentRef)
```

Comment:
there is a `getPayment` request on line 189 after the call to `logResult`, so the customer-perceived response metric may not be accurate and include the full capture path.

Should we call end at the entry `ForageSDK.kt` level and set a boundary around the beginning of the `checkBalance` method and the time when the client gets a response.

In this case, we also wouldn't need to worry about drilling monitors through numerous request methods